sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -352,6 +352,7 @@ def set_authorizer(cls, authorizer: str):
     force_gossip_topology_changes: bool = None  # False
 
     reader_concurrency_semaphore_cpu_concurrency: int = None
+    join_ring: bool = None,  # False is default
```

Comment:
still SCT must be aware of zero token nodes, yaml param is not enough. See my proposal to have them as separate pool instead of adding them to `cluster.nodes`

Reply:
I agree with @soyacz idea of separation.

Reply:
Implementation is next:
 - nodes - pool of all nodes in sct
 - data_nodes - property with only data_nodes
 - zero_nodes - property with only znodes
Nemesis use target only from data_node pool
I go over all places and switch to use correct pool of nodes